modules/installation-and-upgrade/pages/container-deployment/uyuni/migrate-uyuni-to-a-container.adoc

```diff
@@ -70,3 +70,10 @@ or
 ----
 mgradm migrate kubernetes <{productname}.source.fqdn>
 ----
+
+[IMPORTANT]
+====
+
+After successfully running the [command]``mgradm migrate` command, the Salt setup on all minions will still point to the old server. To redirect them to the new server, it is required to use the same FQDN and IP address for the new server as the old one.
```

Comment:
```suggestion
After successfully running the [command]``mgradm migrate`` command, the {salt} setup on all clients will still point to the old server (source server).
To redirect them to the new server (destination server), it is required to use the same FQDN and IP address for the new server as the old one.
```

Reply:
same domain name

Reply:
"source server" does not make it clearer imho

important is 4.3 and 5.0

Reply:
I agree with you.  Unfortunately, we intro'ed and approved the other terms at the beginning of this file:

> "The original server is referred to as the source server, while the newly set-up machine is designated as the destination server."

@jcayouette I'd rather vote to get rid of "source" and "destination" here and use old resp. version number (4.3) and new resp. version number (just 5, maybe {productnumber}) instead.  Version numbers are save because for uyuni we have a separate file...

Reply:
Eric of course is correct. Agreed lets use version numbers.